qiskit/visualization/timeline/events.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""
+Bit event manager for scheduled circuits.
+
+This module provides a `BitEvents` class that manages a series of instructions for a
+specific circuit bit. Bit-wise filtering of the circuit program makes the arrangement of bit
+easier in the core drawer function. The `BitEvents` class is expected to be called
+by other programs (not by end-users).
+
+The `BitEvents` class instance is created with the class method ``load_program``:
+    ```python
+    event = BitEvents.load_program(sched_circuit, qregs[0])
+    ```
+
+Loaded circuit instructions are saved as ``ScheduledGate``, which is a collection of instruction,
+associated time, and bits. All gate instructions are returned by `gates` method.
+Instruction types specified in `BitEvents._non_gates` are not considered as gates.
+If the instruction is associated with multiple bits and the target bit of the class instance is
+the primary bit of the instruction, the instance also generates a ``GateLink`` object
+that shows a relationship between bits during the multi-bit gates.
```

Comment:
```suggestion
Bit event manager for scheduled circuits.

This module provides a `BitEvents` class that manages a series of instructions for a
specific circuit bit. Bit-wise filtering of the circuit program makes the arrangement of bits
easier in the core drawer function. The `BitEvents` class is expected to be called
by other programs (not by end-users).

The `BitEvents` class instance is created with the class method ``load_program``:
    ```python
    event = BitEvents.load_program(sched_circuit, qregs[0])
    ```

Loaded circuit instructions are saved as ``ScheduledGate``, which is a collection of instruction,
associated time, and bits. All gate instructions are returned by the `gates` method.
Instruction types specified in `BitEvents._non_gates` are not considered as gates.
If the instruction is associated with multiple bits and the target bit of the class instance is
the primary bit of the instruction, the instance also generates a ``GateLink`` object
that shows the relationship between bits during multi-bit gates.
```